CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 - PR #161 Use `std::atexit` to finalize RMM after Python interpreter shutdown
 - PR #165 Align memory resource allocation sizes to 8-byte
+- PR #175 Drop `cython` from run requirements
```

Comment:
This should be under bug fixes or improvements. It's not a new feature.

Reply:
This is under improvements.